com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -847,22 +845,15 @@ var Module = {
         Module._isEngineLoaded = true;
         Module.setupCanvas(appCanvasId);
 
-        Module.arguments = [];
-        for(let arg of CUSTOM_PARAMETERS["engine_arguments"])
-            Module.arguments.push(arg);
-        if(window.location.search) {
-            const params = new URLSearchParams(window.location.search);
-            for (const [key, value] of params)
-                Module.arguments.push(`--${key}=${value}`);
-        }
+        Module.arguments = CUSTOM_PARAMETERS["engine_arguments"];
```

Comment:
Revert previous changes.

Reply:
I'll think how to reintroduce it later. But for next beta it should be reverted because one user tested "nightly" build and it was problem with running game because of passing all arguments from url.

Reply:
Maybe we can introduce a special argument just for that or somet### ng like this?